docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,148 @@
+# Developing Themes for Accessibility
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Using alt text](#using-alt-text)
+- [Text accessibility](#text-accessibility)
+- [Keyboard accessibility](#keyboard-accessibility)
+- [Customization](#customization)
+- [Resources](#resources)
+</div>
+
+Designing for accessibility means to be inclusive to the needs of all users. In this tutorial, you will learn how to implement accessibility in your theme and improve theme usability for all users.
+
+We have included four accessibility best practices and examples that will help you develop a theme that meets the needs of the broadest range of abilities.
+
+Prerequisites
+* [A BigCommerce store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial)
+* This tutorial uses Cornerstone. Your theme may differ.
+
+## Using alt text
+Alternative (alt) text is an image description read aloud by a screen reader, allowing the content to be accessible to visually impaired users. Developers should use the aria-label attribute, a [WCAG (Web Content Accessibility Guidelines)](https://www.w3.org/WAI/standards-guidelines/wcag/) technique, to ensure merchants provide alt text for all image elements.
+
+The aria-label attribute uses an ID reference value that matches the ID element attribute to associate an element with text. Screen readers use the text of one or more identified elements, which can be seen elsewhere on the page.
+
+
+Example
+
+The example below displays how to use an aria-label for two elements considered to be one image.
+
+```json
+<div role="img" aria-label="Cart images">
+
+<img src = "giftCard.png" alt=""/>
+<img src = "Checkout.png" alt=""/>
+</div>
+```
+
+You can add the alt text to the `img` element in the image widget of Page Builder or in the `cornerstone/assets/img/` directory.
+
+Follow the step below to add alt text to an image using the control panel:
+1. Go to **Storefront** > **My theme** in the control panel, then click **Customize**.
+2. Launch Page Builder and select the image widget.
+3. Enter the alt text.
+4. Click **Save**.
+
+![Image Widget](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_image_widget.png "Image Widget")
+
+## Text accessibility
+Text accessibility involves making content readable and understandable. You can achieve text accessibility by using the following:
+* Color contrast
+* Text headings
+* Text size and alignment
+* Text links
+
+### Color contrast
+Color contrast is an essential factor in visual accessibility. Use contrasting colors for better readability, along with keeping headings clear and concise.
+
```

Comment:
NOTE: According to [WCAG 2.1 (Web Content Accessibility Guidelines)](https://www.w3.org/TR/WCAG21/), your eCommerce website must have a minimum color contrast ratio of at least 4.5.1 for standard text and 3:1 for large text.